draft-wood-icnrg-ccnxkeyexchange-01.md

```diff
@@ -178,19 +178,45 @@ and producer using message encapsulation.
 
 DISCLAIMER: This is a WIP draft of CCNxKE and has not yet seen rigorous security analysis.
 
-Ephemeral sessions similar to those enabled by TLS 1.3 {{TLS13}},
-QUIC {{QUIC}}, and DTLS 1.2 {{DTLS12}} are needed for some
-CCN exchanges between consumers and producers. Currently, there does not exist
-a standard way to establish these sessions. Thus, the primary goal of
-the CCNxKE protocol is to provide privacy and data integrity between two CCN-enabled
-peers (e.g., a consumer and producer engaged in session-based communication). It
+CCNx Key Exchange (CCNxKE) establishes ephemeral forward secure keys between two peers,
+called the consumer (client) and producer (server).  The underlying cryptography of
+CCNxKE is similar to TLS 1.3, though there are some protocol changes due to the ICN nature
+of CCNxKE.  CCNxKE also supports the concept of a MoveToken, which allows then authenticating
+producer to shift a session to one (or more) co-operating replicas.  
+
+CCNxKE does not specify how the keys are used.  It only specifies how to derive the traffic
+secret that could be used to encrypt/decrypt data.  The draft [draft-wood-icnrg-tlvencap] specifies
+one way to use the traffic secret to carry out communications.  
+Annex A also sketches out an example CCNx protocol for exchangig encrypted messages,
+though it is not part of this standard.
+Other protocols may use CCNxKE.  
+For example, a producer and replica may use CCNxKE to establish a shared key to use
+in Move Tokens.  Two routers may use CCNxKE to establish MACSEC keys.  A consumer and publisher could establish
+a symmetric key while on-line then publish content later for an off-line consumer.
+The use of CCNxKE does not, in short, require the use of a TLS-like transport protocol.
```

Comment:
"In short, the use of CCNxKE is not limited to a TLS-like secure transport protocol."